lib/ClangImporter/ImportDecl.cpp

```diff
@@ -4927,6 +4935,12 @@ namespace {
             dc, decl, decl->parameters(), decl->isVariadic(),
             isInSystemModule(dc), &bodyParams, importedName,
             asyncConvention, errorConvention, kind);
+
+        if (!importedType) {
+          Impl.addImportDiagnostic(
```

Comment:
Why not put this on line 4931/4945?

Reply:
Looks appropriate. At the time I wasn't trying to diagnose property accessors, and to be honest I haven't used Objective-C enough to know precisely what they are. I can guess but I'll do a little reading and move it if it seems reasonable.

Reply:
Ah, sorry, didn't see what visitor this was. Yeah no need to update it. That can be done after the fact if it's easier.